client/scripts/MAPROOM.as

```diff
@@ -126,6 +128,10 @@ package
                         ShowB();
                      }
                   }
+                  else if(!_initFinishedMR)
+                  {
+                     GLOBAL.Message(KEYS.Get("The Maproom is still initializing. Please wait a moment before opening it."));
```

Comment:
Can we change this hardcoded text to come from the language file located in `server/public/gamestage/assets/en.v8.json`.

Let's create a new key with this name and this text on line 1648, just before `newmap_bm_name`:

```json
"newmap_init_setup": "Please wait a moment while we set up the MapRoom. This should only take a few seconds"
```

So now you can call `newmap_init_setup` in your message, rather than the hardcoded text.

Reply:
> Can we change this hardcoded text to come from the language file located in `server/public/gamestage/assets/en.v8.json`.

Yep!
I've made the change you requested. It should be working now!
I left it hardcoded initially from not knowing where the language files were, apologies.

Reply:
Great, thanks! Just one last thing, can we add a comma to the end of that JSON that you added in `en.v8.json`, otherwise it's not valid JSON. ^^